### PR TITLE
Test all stacks

### LINKED ISSUE #20: Add testing for all stacks
Heroku-20 beta was just released:
https://devcenter.heroku.com/changelog-items/1937

I tried installing this buildpack on a Heroku-20 app, and the build succeeded, though I didn't test any further.

It would be great to add some per-stack testing to this buildpack, to ensure it works on Heroku-20 at runtime and to prevent future stack-specific regressions like #9).

Many thanks :-)

### PR DESCRIPTION
**Pull request by dzuelke**
Closes #20

GUS-W-8276311

### CODE REVIEW

**Comment by dzuelke** [comment]
Once ready, **please merge this after #31 and #32**

**Review by edmorley — APPROVED** [review]

**Pull request merged**
